web/source/kmwembedded.ts

```diff
@@ -256,10 +256,7 @@ namespace com.keyman.text {
    */
   keymanweb['correctOSKTextSize']=function() {
     let osk: com.keyman.osk.AnchoredOSKView = keymanweb.osk;
-    if(osk?.vkbd) {
-      osk._Load(); // TODO:  replace with osk.refreshLayout() in the future once it can perfectly
-                   //        handle rotations.
-    }
+    osk.refreshLayout();
```

Comment:
```suggestion
    keymanweb.osk?.refreshLayout();
```

The previous code used optional chaining for `osk` -- could `osk` ever be nullish?

Also, we might as well dump the local variable here when this is a one-liner.

Reply:
Note that the 'optional chaining' used by 'the previous code' was based on whether or not the `VisualKeyboard` component of the OSK had been initialized.  Not whether or not the main `OSKView` itself was nullish.

Reply:
The old code did a `osk?.`:

```
  if(osk?.vkbd) {
```

Which would be `false` if `osk` is nullish. So from what I can see, it did test whether or not the main `OSKView` itself was nullish?